### PR TITLE
thelio-r2 audio fixup

### PR DESCRIPTION
**Pull request by jackpot51**
Rebase and make release when approved

### CODE REVIEW

File: system76driver/daemon.py [hunk 1]
```diff
@@ -653,6 +658,20 @@ def run_headphone_volume_adjust(model):
...
 
+def _run_line_volume_toggle(model):
+    if model not in NEEDS_LINE_VOLUME_TOGGLE:
+        log.info('Line volume toggle not needed for %r', model)
+        return
+    log.info('Line volume toggle for %r', model)
+    subprocess.call(["amixer", "-c", "Generic", "set", "Line", "mute"])
+    subprocess.call(["amixer", "-c", "Generic", "set", "Line", "unmute"])
+
+def run_line_volume_toggle(model):
```
Comment (ids1024) [inline system76driver/daemon.py:669]: Unless there's some particularly arcane Python I'm missing, this function doesn't seem to be called.

Comment (jackpot51) [reply system76driver/daemon.py:669]: Oh duh

**Review by ids1024 — DISMISSED** [review]

**Comment by leviport** [comment]
Dang, it's not working even though I see the "Line volume toggle for thelio-r2" message in logs. I assume since system76-driver is running it, the action is being run with elevated privileges? The fix I had working was running the commands from a shell script with regular user permissions (I scheduled it with Startup Applications).

**Comment by leviport** [comment]
It seems to be more issues with amixer needing the commands in goofy sequences, as well as the Line channel starting at 0% volume by default. I'm going to futz with it some more, and I think I can tweak it to make it work.

**Comment by ids1024** [comment]
I believe since PulseAudio runs as the user rather than root, this would also run before pulseaudio starts.

**Comment by leviport** [comment]
I'm not sure, it seems like I got it working reliably after running `amixer -c 1 set Line 70%` as the user just one time. The trick will be finding out whether or not it works from a clean install. I'll make a branch off this one to fool around with if it looks promising.

**Comment by jackpot51** [comment]
PulseAudio runs after this and probably re-breaks it. If the issue is the Audio device going into RTD3, maybe we can disable that instead.

**Review by ids1024 — APPROVED** [review]
This seems like a better solution (if it works).

**Review by leviport — APPROVED** [review]

**Pull request merged**